plugins/woocommerce-admin/babel.config.js

```diff
@@ -23,28 +23,6 @@ module.exports = function ( api ) {
 		ignore: [ 'packages/**/node_modules' ],
 		env: {
 			production: {},
-
-			storybook: {
```

Comment:
Storybook doesn't use this config because we've moved the storybook to `tools/storybook/`.